resources/langs/nheko_de.ts

```diff
@@ -634,11 +634,16 @@
     </message>
     <message>
         <location line="+2"/>
-        <source>Unbanned %1</source>
-        <translation>Hat die Verbannung von %1 zurückgezogen.</translation>
+        <source>Unbanned %1.</source>
+        <translation type="unfinished">Hat die Verbannung von %1. zurückgezogen.</translation>
```

Comment:
The dot in the german translation is wrong, it should only be in the end. (I don't think you know german, so you probably don't know?)